src/main/webapp/app/instructor-exercise-dashboard/instructor-exercise-dashboard.component.ts

```diff
@@ -0,0 +1,78 @@
+import {JhiAlertService} from 'ng-jhipster';
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {HttpResponse, HttpErrorResponse} from '@angular/common/http';
+import {Exercise, ExerciseService} from 'app/entities/exercise';
+import {InitializationState, Participation, ParticipationService} from 'app/entities/participation';
+import {Result, ResultService} from 'app/entities/result';
+import {TutorLeaderboardData} from 'app/instructor-course-dashboard/tutor-leaderboard/tutor-leaderboard.component';
+
```

Comment:
the formatting is not correct, please use import { JhiAlertService } from 'ng-jhipster';

--> make sure to change your IntelliJ settings for this space

Reply:
Done